src/Components/Login.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import { signInUser } from '../state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
+import LoginForm from './LoginForm'
+
+class Login extends Component {
+  state = {
+    email: '',
+    username: '',
```

Comment:
Where do you set the state for username?

Reply:
Removed